### PR TITLE
Feature/add participants

### PR DESCRIPTION
**Pull request by lprasol**

### CODE REVIEW

**Comment by radowit** [comment]
What I did to fix Hasura:
- removed nb_meeting_participants from all migrations and metadata (and removed migrations that contained only this)
- restarted docker
- re-added meeting participant table and relationships

File: app/notice_board/views.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                participant.status = MeetingParticipant.CONFIRMED
+                status = participant.status
+                participant.save()
+        return Response({"status": status.lower()})
+
+    # pylint: disable=unused-argument
+    @action(detail=True, methods=["post"])
+    def remove_participant(self, request: Request, pk=None):
+        meeting = self.get_object()
+        meeting.participants.remove(request.user)
```
Comment (radowit) [inline app/notice_board/views.py:46]: Atfer you remove a participant you need to check i there is anyone that can pop in for his place.
You could reuse code from add participant action.

**Review by radowit — CHANGES_REQUESTED** [review]

File: app/notice_board/views.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        )
+        status = MeetingParticipant.WAITING
+        participants = MeetingParticipant.objects.filter(
+            meeting=meeting,
+        ).order_by("created_at")
+        for i, participant in enumerate(participants):
+            if i >= meeting.participants_limit:
+                break
+            if participant.user == request.user:
+                participant.status = MeetingParticipant.CONFIRMED
```
Comment (radowit) [inline app/notice_board/views.py:37]: This branch is not covered in unit tests

**Review by radowit — APPROVED** [review]

**Comment by lprasol** [comment]
[Zapisywanie się na sesje](https://trello.com/c/QUxSZ942/39-zapisywanie-si%C4%99-na-sesje)

**Pull request merged**